src/components/Overview/Overview.tsx

```diff
@@ -0,0 +1,44 @@
+import { ReactElement } from 'react';
+import { PageNode } from 'src/directory/directory';
+import { Card, Flex, Grid, View, Heading } from '@aws-amplify/ui-react';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+type OverviewProps = {
+  childPageNodes: PageNode[];
+};
+
+export function Overview({ childPageNodes }: OverviewProps) {
+  const router = useRouter();
+  const currentPlatform = router.query.platform;
+
+  if (!childPageNodes) {
+    return <></>;
+  }
+
+  return (
+    <Grid className="overview">
+      {childPageNodes.map((node) => (
+        <Link
+          key={node.route}
+          className="overview__link"
+          href={{
+            pathname: node.route,
+            query: { platform: currentPlatform ? currentPlatform : '' }
+          }}
+        >
+          <Card className="overview__link__card" variation="outlined">
+            <Flex direction="column">
+              <Heading level={3} className="overview__link__card__title">
```

Comment:
These will only work as an h3 (accessibility-wise) if we can guarantee there is an h2 above them. So we could add a static to heading this component that is an h2 that is like "In this section" or something, or I would make these `<Text as="span" className="overview__link__card__title"></Text>`